README.md

```diff
@@ -0,0 +1,43 @@
+# CAOTIC
+High Contrast and Adaptive Optics laboratory testbeds community website
+
+Welcome to the community website for astronomical testbeds! We are currently working on getting our 
+website migrated to highconaotools.github.io, the original website can be found here:  
+https://sites.google.com/view/highcontrastlabs/
+
+## Local setup - MacOS
+
+### Initial setup
+
+If you use conda, make sure you have deactivated all your environments (including `base`) by doing `$ conda deactivate` often enough.
+
+*On a Mac*:
+1. Install homebrew following the instructions on this page:  
+https://osxdaily.com/2018/03/07/how-install-homebrew-mac-os/
+
+2. Install the package manager rbenv (more on rbenv [here](https://github.com/rbenv/rbenv) ) 
+`$ brew install rbenv`
+
+3. Set up rbenv in your shell  
+`$ rbenv init`
+
+4. Close and reopen your terminal for the changes to take effect
+
+5. Install ruby (more about using rbenv on a Mac [here](https://medium.com/@norton.seanm/a-guide-to-using-rbenv-mac-4ca211b1c713))  
+`$ rbenv install 2.7.2`
+
+6. Set your local ruby version to use  
+`$ rbenv local 2.7.2`
+
+7. Install bundler and jekyll  
+`$ gem install bundler jekyll`
+
```

Comment:
Need to also install `jemoji` to get `jekyll serve` to do anything.

Reply:
Now the question is, can we do that all in the same line...

Reply:
Probably! This was the full error, in case this is helpful: `Dependency Error: Yikes! It looks like you don't have jemoji or one of its dependencies installed. In order to use Jekyll as currently configured, you'll need to install this gem. If you've run Jekyll with `bundle exec`, ensure that you have included the jemoji gem in your Gemfile as well. The full error message from Ruby is: 'cannot load such file -- jemoji' If you run into trouble, you can find helpful resources at https://jekyllrb.com/help/!`

Reply:
In my side i had to install jemoji. I will do all these steps again in another computer with ubuntu to add a part for ubuntu OS if it's not already done.

Reply:
@ivalaginja have you tested that putting them all on one line works?

Reply:
I think Kevin did, and it worked